core/communication/bubblewrap/src/main/javascript/index.ts

```diff
@@ -208,6 +208,7 @@ export class Bubblewrap {
      * Creates a new instance of [[Bubblewrap]] with the specified dictionary of registered classes.
      *
      * If no dictionary is specified, no classes are registered.
+     * Classes will need to be added later via [[addClasses]]
```

Comment:
It's not entirely clear to me the point of having a private constructor here. If it's done to enforce a singleton, that's not done either. So I really have no idea. There's also a `codec` instance variable that's not set, by default, in the constructor.